redisson-hibernate/redisson-hibernate-52/src/main/java/org/redisson/hibernate/RedissonRegionFactory.java

```diff
@@ -167,6 +179,28 @@ public long nextTimestamp() {
                 + "redis.call('set', KEYS[1], nextValue); "
                 + "return nextValue;",
                 RScript.ReturnType.INTEGER, Arrays.<Object>asList("redisson-hibernate-timestamp"), time);
+        } catch(Exception e) {
+            if(fallback) {
+                return fallbackNextTimestamp();
```

Comment:
use `SimpleTimestamper.next()` instead

Reply:
`SimpleTimestamper` isn't available in Hibernate 5.2.  

Hibernate 5.2 has this logic in `org.hibernate.cache.jcache.time.Timestampe`, under `hibernate-jcache`, but I'm assuming we wouldn't want to add `hibernate-jcache` as a dependency here, correct?  

Please advise - I'm glad to change, however you deem appropriate. 

Thanks!

Reply:
There is a problem - we can't get LGPL 2.1 code and put it in project under Apache License.

Reply:
Can we add this dependency then, in redisson-hibernate-52's pom.xml?
```
       <dependency>
         <groupId>org.hibernate</groupId>
         <artifactId>hibernate-jcache</artifactId>
         <version>5.2.18.Final</version>
         <optional>true</optional>
       </dependency>
```

And then call `Timestamper.next()`?

Thanks.

Reply:
I need to think how to make it better